hedera-node/src/main/java/com/hedera/services/sigs/HederaToPlatformSigOps.java

```diff
@@ -119,24 +116,24 @@ public static SignatureStatus expandIn(
 	 *     processing and return a {@link SignatureStatus} representing this error.</li>
 	 * </ul>
 	 *
-	 * @param txnAccessor the accessor for the platform txn.
-	 * @param syncVerifier facility for synchronously verifying a cryptographic signature.
-	 * @param keyOrderer facility for listing Hedera keys required to sign the gRPC txn.
-	 * @param sigsProvider source of crypto sigs for the simple keys in the Hedera key leaves.
+	 * @param txnAccessor the accessor for the platform txn
+	 * @param syncVerifier facility for synchronously verifying a cryptographic signature
+	 * @param keyOrderer facility for listing Hedera keys required to sign the gRPC txn
+	 * @param pkToSigFnProvider source of crypto sigs for the simple keys in the Hedera key leaves
```

Comment:
nit: `@param...`  for `sigFactoryCreator` is missing

Reply:
Added!